tests/execution_engine/test_pandas_execution_engine.py

```diff
@@ -1038,7 +1040,7 @@ def test_constructor_with_gcs_options(mock_gcs_conn, mock_auth_method):
     custom_gcs_options = {"filename": "a/b/c/my_gcs_credentials.json"}
     engine = PandasExecutionEngine(gcs_options=custom_gcs_options)
     assert "gcs_options" in engine.config
-    assert "filename" not in engine.config.get("gcs_options")
+    assert "filename" in engine.config.get("gcs_options")
```

Comment:
I believe this was a bug. We are testing whether we can load a custom GCS option into the ExecutionEngine configuration, we would want to test that it got in.. rather than check if it didn't

Reply:
@Shinnnyshinshin Any clue as to why this "wrong" test worked before?..

Reply:
Thank you for bringing this up. I had a closer look at the code, and updated the previous comment with a more accurate understanding

Reply:
❤️